### DISCUSSION TITLE
Discord community call: 17 August 2024

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#6866] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: August 17th, 2024 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/aug-17-2024/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- TBD

**Comment by BenJamesBen** [comment, marked as answer]
### August 17, 2024 Community Call

#### Notes:
- What projects people are working on.
- The `ringsector` macro issue. Benjamin F. will work with bobgeorgethe3rd to get a list of the Points `ringsector` should provide, get them added to the documentation, and then get those Points re-added/reverted in `ringsector` code. Benjamin F. will report back next meeting with an update.
- Should we have a code review process for changes to prevent issues like this?
- Is anyone working on a design that has a kimono-style, built-in, all-in-one sleeve?

#### Attendees:
- Benjamin F.
- bobgeorgethe3rd
- Curvalicia (a.k.a. Anna Puk)
- Darigov Research
- RockerKitten
- TangerineShark
- Thrunic

**Reply by darigovresearch** [reply]
Thanks for the notes @BenJamesBen !

Additional links for reference:
- Have made a discussion point for a kimono design - https://github.com/freesewing/freesewing/discussions/7014
- Have also made a discussion point for gi design - https://github.com/freesewing/freesewing/discussions/7015
